src/tools/PICS-generator/PICSGenerator.py

```diff
@@ -419,7 +419,7 @@ async def test_device_mapping(self):
         if args.dm_xml:
             xml_clusters, problems = build_xml_clusters(Path(f"{args.dm_xml}/clusters"))
         else:
-            xml_clusters, problems = build_xml_clusters(PrebuiltDataModelDirectory.kMaster)
+            xml_clusters, problems = build_xml_clusters(PrebuiltDataModelDirectory.k1_5)
```

Comment:
could you add a comment on why a specific version is better than Master?
Does this place us on an update treadmill (every release we have to bump this up?)

Reply:
The intention here was to fallback to master if the --dm parameter is not passed, so I'm a bit on the fence on using any specific version here 🤔 I did notice it failed without the --dm tag, and I wanted to dig into it once TE#2 had ended. But honestly I would not do this change.

Reply:
As an alternative I raised https://github.com/project-chip/connectedhomeip/pull/39021

Reply:
I can close this one if you want - it was just a quickie fix so folks could use the tool for TE2.

Reply:
I have changed #39021 to do exactly what is requested in the issue 😊